### PR TITLE
Update to latest metrics, unblock collector upgrade

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: exporters/metric/cortex/cortex.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/golang/snappy"
 	"github.com/prometheus/prometheus/prompb"
 
+	"go.opentelemetry.io/otel/sdk/resource"
+
```
Comment (pellared) [inline exporters/metric/cortex/cortex.go:31]: nit: can you remove this line and group the OTel imports together?

File: exporters/metric/cortex/cortex_test.go [hunk 2]
```diff
@@ -31,12 +31,13 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+
```
Comment (pellared) [inline exporters/metric/cortex/cortex_test.go:36]: nit: can you remove this line and group the OTel imports together?

File: exporters/metric/cortex/testutil_test.go [hunk 3]
```diff
@@ -15,15 +15,21 @@
...
 	"testing"
+	"time"
 
 	"github.com/prometheus/prometheus/prompb"
 	"github.com/stretchr/testify/require"
 
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/sdkapi"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+
```
Comment (pellared) [inline exporters/metric/cortex/testutil_test.go:29]: nit: can you remove this line and group the OTel imports together?

File: exporters/metric/datadog/datadog.go [hunk 4]
```diff
@@ -21,6 +21,8 @@ import (
 
 	"github.com/DataDog/datadog-go/statsd"
 
+	"go.opentelemetry.io/otel/sdk/resource"
+
```
Comment (pellared) [inline exporters/metric/datadog/datadog.go:25]: nit: can you remove this line and group the OTel imports together?

File: exporters/metric/dogstatsd/dogstatsd_test.go [hunk 5]
```diff
@@ -17,18 +17,24 @@ package dogstatsd_test
...
+	"time"
 
 	"github.com/stretchr/testify/require"
 
+	"go.opentelemetry.io/otel/metric/sdkapi"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+
```
Comment (pellared) [inline exporters/metric/dogstatsd/dogstatsd_test.go:32]: nit: can you remove this line and group the OTel imports together?

File: exporters/metric/dogstatsd/internal/statsd/conn_test.go [hunk 6]
```diff
@@ -17,30 +17,33 @@ package statsd_test
...
+	"sync"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 
+	"go.opentelemetry.io/otel/metric/sdkapi"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+
```
Comment (pellared) [inline exporters/metric/dogstatsd/internal/statsd/conn_test.go:33]: nit: can you remove this line and group the "3rd party" imports together?

**Review by pellared — APPROVED** [review]
👍

File: exporters/metric/cortex/cortex.go (hunk 1 above)
Comment (bogdandrutu) [reply exporters/metric/cortex/cortex.go:31]: No order in these files, so not sure it matters.

Comment (pellared) [reply exporters/metric/cortex/cortex.go:31]: it is just a "nice to have". as for me: feel free to ignore my comments and thanks a lot for your PR 🎉 😉

Comment (bogdandrutu) [reply exporters/metric/cortex/cortex.go:31]: I think we should enforce an order and fix all the files.

Comment (bogdandrutu) [reply exporters/metric/cortex/cortex.go:31]: Happy to do that in a separate PR.

**Comment by jmacd** [comment]
Please use https://github.com/open-telemetry/opentelemetry-go-contrib/pull/1091

**Pull request closed without merging**